source/agent/addons/quic/QuicTransportStream.cc

```diff
@@ -214,6 +221,19 @@ NAN_SETTER(QuicTransportStream::isMediaSetter)
     obj->m_isMedia = value->ToBoolean(Nan::GetCurrentContext()).ToLocalChecked()->Value();
 }
 
+NAN_GETTER(QuicTransportStream::ondataGetter)
```

Comment:
Do you need register `ondataGetter` somewhere like `init` function?

Reply:
Added.